estate/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Application"
+
+    name=fields.Char(string="Name",required=True)
+    description=fields.Text(string="Description")
+    postcode=fields.Char(string="Postcode")
+    date_availability=fields.Date(string="Availability",copy=False,default=fields.Date.add(fields.Date.today(),days=90))
```

Comment:
The date_availability should have a default of 3 months, not 90 days
```suggestion
    date_availability=fields.Date(string="Availability", copy=False, default=fields.Date.add(fields.Date.today(), months=3))
```